core/src/thrift/tachyon.thrift

```diff
@@ -148,7 +148,7 @@ service MasterService {
    * return the command from master to worker. addedBlockIds maps from id of storage directory
    * to the blocks added in it.
    */
-  Command worker_heartbeat(1: i64 workerId, 2: i64 usedBytes, 3: list<i64> removedBlockIds,
+  Command worker_heartbeat(1: i64 workerId, 2: list<i64> usedBytesByAlias, 3: list<i64> removedBlockIds,
```

Comment:
How about rename usedBytesByAlias to usedBytesOnTiers? 
And different  workers may have different hierarchy configurations,  how to make sure used byes in the list(usedBytesByAlias) mapping to the proper layer?  eg: 
worker0:  layer0: SSD    layer1: HDD  
worker1:  layer0: MEM   layer1: SSD    layer2: HDD